include/swift/SILOptimizer/PassManager/PassPipeline.h

```diff
@@ -56,7 +56,9 @@ class SILPassPipelinePlan final {
   SILPassPipelinePlan() = default;
   ~SILPassPipelinePlan() = default;
   SILPassPipelinePlan(const SILPassPipelinePlan &) = default;
+#if !defined(_MSC_VER) || defined(__clang__)
   SILPassPipelinePlan(SILPassPipelinePlan &&) = delete;
```

Comment:
Yeah, this has been fixed BTW: https://connect.microsoft.com/VisualStudio/feedback/details/3116505/msvc-fails-to-compile-code-that-compiles-with-clang-reports-attempting-to-reference-a-deleted-function-error-from-destructor